src/lib/config.js

```diff
@@ -30,7 +30,7 @@ class Config {
     this.timezone = 0;
 
     this.locale = 'enUS';
-    this.os = 'Mac';
+    this.os = 'Win';
```

Comment:
I think separating the commit, but including it in the same PR would be great. I'm mostly thinking about what happens to tools like git blame when commits do too many separate things. :) @timkurvers What do you think?